Dev/src/screens/TeacherScreens/ClassTabs/ClassMainScreen.js

```diff
@@ -88,6 +83,16 @@ export class ClassMainScreen extends QcParentScreen {
     );
 
   }
+//to write implemntation of the function, updates class name
+updateTitle(newTitle){
+  this.setState({titleHasChanged: true})
+  this.setState({currentClass: {...this.state.currentClass, name: newTitle}})
+}
+updatePicture(newPicture){
+  this.setState({pictureHasChanged: true})
+  this.setState({currentClass: {...this.state.currentClass, name: newPicture}})

```

Comment:
Set the profileProfileImageID to the picture, not the name of the class. Otherwise, this should work